strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/exception/SizeFormatValidationException.java

```diff
@@ -0,0 +1,10 @@
+package org.carlspring.strongbox.exception;
+
+public class SizeFormatValidationException extends RuntimeException {
+
+    public SizeFormatValidationException(final String message)
+    {
+        super(message);
+    }
+
+}
```

Comment:
Cosmetic: Please, add a new line at the end of the file.

Reply:
New line added